usr/lib/cca_stdll/cca_specific.c

```diff
@@ -1742,6 +1719,18 @@ static CK_RV cca_get_adapter_domain_selection_infos(STDLL_TokData_t *tokdata)
     }
     TRACE_DEVEL("num_adapters: %u\n", cca_private->num_adapters);
 
+#if defined(_AIX)
+   /*
+    * Short-circuit! If we're on Power, it means the card does not support CCA
+    * DOMAINs. Therefore we force a single domain and skip all future tests.
+    *
+    * Future commits might remove conditional compilation if Z-hosted cards are
+    * enabled to be used remotely on AIX.
+    */
+    cca_private->num_usagedoms = 1;
+    cca_private->num_domains = 1;
+    return CKR_OK;
```

Comment:
I would remove the `return CKR_OK;` from within the ifdef block. There is one after the else block. 

If one adds code to this function in the future after the else block, then it might be overseen that it returns already here on AIX.